### DISCUSSION TITLE
partnership

### DISCUSSION THREAD

**Discussion by RaeAtBiscience** [Cookie-AutoDelete/Cookie-AutoDelete#1583] [General]
Hello Kenneth! Were looking to partner with you - please email me to: rae {at} biscience.com

**Comment by kennethtran93** [comment]
> Hello Kenneth! Were looking to partner with you - please email me to: rae {at} biscience.com

I'm not sure what we can be partners for - this is an open source webextension, where anybody can help put in fixes and whatnot, and I only have a small amount of time to work on this from time to time.

**Comment by RaeAtBiscience** [comment]
The partnership is for analytics from the extension.
We can pay you a monthly revenue for your extensions' analytics on a csv file, for our internal product (the analytics are anonymous).

Please email me directly and we can schedule a call to further discuss.

________________________________

**Comment by kennethtran93** [comment]
I'm not sure what analytics you intend to get from us.  The web extension does not send any analytics anywhere.  The only analytics we have are what's visible to us as the extension developers/publishers in each of the browser extension stores.

While I appreciate you reaching out, providing any analytics from the extension to third parties is not and will not be part of our terms or privacy policy.

**Comment by RaeAtBiscience** [comment]
We buy anonymous clickstream analytics from extension developers.
We buy it for our internal product. You would have to extract it into csv files on a monthly basis.
If you'd like to sell to us your analytics, please feel free to reach out to me: ***@***.***
________________________________
